config/windows-containerd-monitor-filelog.json

```diff
@@ -20,6 +20,11 @@
 			"type": "temporary",
 			"reason": "IncompatibleContainer",
```

Comment:
Lastly, we should consider removing this stop gap condition since we have the corrupt image one.